### PR TITLE
WIP: Move LimitRange Examples Out of no-ci Folder

### LINKED ISSUE #2071: Add e2e Test for Having a LimitRange
#2020 added the ability for TaskRuns to apply a LimitRange minimum for container resource requests if a LimitRange container minimum in present in the namespace where the TaskRun is running. There should be a test for running a PipelineRun/TaskRun in a namespace with a LimitRange container minimum specified.

This would require a namespace to be created and adding a LimitRange to that namespace. Both a PipelineRun and independent TaskRun should be able to run and succeed while running in the namespace with the LimitRange.

### PR DESCRIPTION
**Pull request by danielhelfand**
Closes #2071 

When these examples were originally introduced, they needed to be located under the `no-ci` folder since the examples created a LimitRange that caused other examples to fail. At the time, the implementation for using LimitRanges required users to specify a `LimitRangeName` on the PipelineRunSpec or TaskRunSpec. No other examples did this, which caused failures during integration tests. All these examples should now be able to succeed even with a LimitRange in place since no LimitRangeName is needed to be specified (#2020).

 I think what makes sense is to namespace these examples as discussed in this [issue in plumbing](https://github.com/tektoncd/plumbing/issues/217) and keep other examples all running in a particular namespace. Not sure if this would require anything different to detect for failures of these examples since they are in different namepaces than other examples, but my thought would be to have these serve as a form of e2e testing for LimitRanges.

# Submitter Checklist

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

```
N/A
```

### CODE REVIEW

File: examples/pipelineruns/limitrange.yaml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: pipelinerun-limitrange-ns
+---
+apiVersion: v1
 kind: LimitRange
 metadata:
   name: limit-mem-cpu-per-container
+  namespace: pipelinerun-limitrange-ns
```
Comment (vdemeester) [inline examples/pipelineruns/limitrange.yaml]: Our "check" won't check this though, as "did it run sucessfully", as, it only look into the current namespace (`default` or one created by the CI :thinking:).

Comment (danielhelfand) [reply examples/pipelineruns/limitrange.yaml]: By chance, do you know if other namespaces could be added to check?

Comment (vdemeester) [reply examples/pipelineruns/limitrange.yaml]: > By chance, do you know if other namespaces could be added to check?

Yep they could :angel:

Comment (danielhelfand) [reply examples/pipelineruns/limitrange.yaml]: Let me look into [e2e-common.sh](https://github.com/tektoncd/pipeline/blob/master/test/e2e-common.sh) and see what I can do to try and add the results.

**Comment by danielhelfand** [comment]
Closing this in favor of adding something to go e2e tests

**Pull request closed without merging**